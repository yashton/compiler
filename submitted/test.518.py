def foo():
	return 2

def bar(x):
	if x == 0:
		return 1
	else:
		return 0

def foobar():
  		if x == 1:
		  y = 2
                else:
		  y = 3
