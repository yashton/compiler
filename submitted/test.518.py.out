(KEYWORD def)
(ID "foo")
(PUNCT "(")
(PUNCT ")")
(PUNCT ":")
(NEWLINE)
(ERROR "Illegal character '	return 2

def bar(x):
	if x == 0:
		return 1
	else:
		return 0

def foobar():
  		if x == 1:
		  y = 2
                else:
		  y = 3
' on line 2)
