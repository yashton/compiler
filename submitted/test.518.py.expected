(KEYWORD def)
(ID "foo")
(PUNCT "(")
(PUNCT ")")
(PUNCT ":")
(NEWLINE)
(KEYWORD return)
(LIT 2)
(NEWLINE)
(KEYWORD def)
(ID "bar")
(PUNCT "(")
(ID "x")
(PUNCT ")")
(PUNCT ":")
(NEWLINE)
(KEYWORD if)
(ID "x")
(PUNCT "==")
(LIT 0)
(PUNCT ":")
(NEWLINE)
(KEYWORD return)
(LIT 1)
(NEWLINE)
(KEYWORD else)
(PUNCT ":")
(NEWLINE)
(KEYWORD return)
(LIT 0)
(NEWLINE)
(KEYWORD def)
(ID "foobar")
(PUNCT "(")
(PUNCT ")")
(PUNCT ":")
(NEWLINE)
(INDENT)
(KEYWORD if)
(ID "x")
(PUNCT "==")
(LIT 1)
(PUNCT ":")
(NEWLINE)
(ID "y")
(PUNCT "=")
(LIT 2)
(NEWLINE)
(INDENT)
(KEYWORD else)
(PUNCT ":")
(NEWLINE)
(ID "y")
(PUNCT "=")
(LIT 3)
(NEWLINE)
(DEDENT)
(DEDENT)
(ENDMARKER)
